_episodes/wrap-up.md

```diff
@@ -0,0 +1,110 @@
+page and their raw content and is often aided by CSS - an ordered set of styling instructions telling the browser 
+how the content from within HTML document should be organised and formatted. It is very verbose and difficult 
+to write by hand - beyond their initial design HTML pages are meant to be processed by machines. 
+For that reason, Markdown was introduced - a lightweight Markup language and a convention for 
+adding style information to textual content while retaining the human-readable form. It is not as rich in syntax as HTML,
+but comparably more usable by a wider audience. 
+
+Jekyll is a powerful static site generator behind GitHub Pages that supports a high degree of reuse and separation of 
+content and presentation. It can be configured via the `_config.yml` YAML file 
+that stores site-wide variables accessible from anywhere within our site. 
+It also allows us to define and reuse page-wide variables, by defining them as the *front matter* at the beginning of
+our Markdown pages. Used in conjunction with Liquid, its scripting language, Jekyll allows us to include content from 
+reusable markup snippets that can be repeated on multiple pages and to embed code snippets and filters into our pages to 
+create some complex content assembly pipelines. That way, adding a new blog post to your website may only involve 
+creating a file in the `_posts` folder of your website, setting it to use the 'post' template and focusing on its 
+content - and it will magically appear at the top of your Blog timeline without any extra effort on your part.
+
+## Jekyll Website Directory Structure
+
+Let's have a look at some directory structures of different Jekyll websites.
+
+First we will visit the directory structure of the [Jekyll website used to develop this lesson](https://github.com/carpentries-incubator/building-websites-with-jekyll-and-github-or-gitlab), which uses the 
+[The Carpentries Jekyll theme](https://github.com/carpentries-incubator/template), and see how it relates to visual components on the website's home page.
+
+![directory-structure-home-page](../fig/directory-structure-home-page-combined-annotated.png) 
+
+Other Jekyll websites will have slightly different structures but there are some common elements, such as `index.md`, 
+`_config.yml`, and folders starting with `_` (such as `_includes` and `_site`) which have special meaning to 
+Jekyll by convention. Other special folders in this case are `_episodes` and `_extras`, but these are defined by the 
+theme designer and Jekyll will not automatically process these. Similar for `fig` folder holding images for the website 
+(which could have been named `images` or `figures` just the same). 
+
+> ## Reusing The Carpentries Jekyll Lesson Theme 
+>Reusing [the lesson template](https://github.com/carpentries-incubator/template) to create a new lesson can be achieved by copying the theme repository and customising it by adding your episodes in the appropriate place. 
```

Comment:
Is this the right link?  I've always used the [styles repo](https://github.com/carpentries/styles) and didn't know this template existed.

Reply:
Think this template is for Incubator lessons used by our lesson too. @tobyhodges will ultimately confirm.

Reply:
@tobyhodges could use your feedback here.

Reply:
There are a few different options at the moment:. As I understand it, the best way to adopt the full lesson template is to import the carpentries/styles repository into a new repo, run `lesson_initialise.py` and start writing the lesson from there. However, the carpentries-incubator/template repository is an easier starting point because it uses the remote theme and some of the lesson initialisation steps have already been taken care of. The downside of the Incubator template for this lesson is that it does not include some components (e.g. the `_layouts` directory) that are discussed in the lesson.

All of this will change for Carpentries lessons when the new template is introduced. For now, I suggest to use the _styles_ repository here.